app/api/v1/endpoints/admin.py

```
@@ -588,6 +653,232 @@ async def list_users(
+        await db.rollback()
+        logger.exception("User verification failed")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="User verification failed"
+        ) from e
+
+
+@router.post("/users/verify-batch")
+async def verify_users_batch(
+    request: BatchVerifyRequest,
+    current_user: User = Depends(require_role([UserRole.ADMIN])),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)